startup.sh

```diff
@@ -71,6 +71,13 @@ function install_docker() {
   yum install -y docker-ce docker-ce-cli containerd.io
 }
 
+function install_docker_rhel() {
+  subscription-manager repos --enable=rhel-7-server-rpms
```

Comment:
who calls subscription-manager register ?

Reply:
the actual rhel host, not the container